apps/frontend/src/components/global/sidebaritems/SidebarFileList.vue

```diff
@@ -74,7 +74,10 @@ export default class FileItem extends mixins(ServerMixin) {
     let file = InspecDataModule.allFiles.find(
       (f) => f.unique_id === this.file.unique_id
     );
-    if (file) {
+    if (file?.database_id){
+      SnackbarModule.failure('This evaluation is already in the databse.')
```

Comment:
Can you have the icon update to be disabled and just have the mouseover text say "Saved" instead? I'd prefer to have a saved with checkbox icon but it seems that icon doesn't actually exist sadly.